.github/workflows-source/ci-main.yml

```diff
@@ -199,6 +182,36 @@ jobs:
         run: |
           sudo rm -rf /private/var/tmp/bazel-output
 
+  bazel-test-macos-namespace:
```

Comment:
Can you add a note about why we run the Darwin jobs twice (on self-hosted + on namespace)?